x-pack/plugins/cases/server/client/cases/client.ts

```diff
@@ -52,7 +52,7 @@ export interface CasesSubClient {
    *
    * If the `owner` field is left empty then all the cases that the user has access to will be returned.
    */
-  find(params: CasesFindRequest): Promise<CasesFindResponse>;
+  search(params: CasesSearchRequest): Promise<CasesFindResponse>;
```

Comment:
![whynotboth](https://github.com/elastic/kibana/assets/1533137/aad608b0-fb70-4209-9b6a-54a84703f99b)

Reply:
We renamed it to `search`, and updated it's schema from `CasesFindRequest ` to `CasesSearchRequest `.
didn't make sense to keep both methods with same functionality.

Reply:
+1